app.py

```diff
@@ -14,10 +14,38 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
+"""solver-license-jon CLI."""
 
-"""This is the main script of the template project."""
+from license_solver import LicenseSolver
+import click
+from typing import Optional
 
-from template.version import __version__
 
-if __name__ == "__main__":
-    print(f"A template project with Thoth integration, v{__version__}.")
+# TODO: pass program arguments
+@click.command()
+@click.pass_context
+@click.option("-f", "--file", type=str, help="Get license from file")
+@click.option("-d", "--directory", nargs=1, type=str, help="Get licenses from folder")
+def cli(
+    _: click.Context,
+    directory: Optional[str] = None,
+    file: Optional[str] = None,
+) -> None:
+    """Parse program arguments."""
+    if directory and file:
+        print("Only one option between directory and file ")
```

Comment:
You can simplify this by using `os.path.isfile` to distinguish these two and have a single option.